tests/FamilyHubs.ServiceDirectory.Api.AcceptanceTests/Tests/Steps/ServiceSearchSteps.cs

```diff
@@ -16,64 +16,86 @@ namespace FamilyHubs.ServiceDirectory.Api.AcceptanceTests.Tests.Steps;
 /// </summary>
 public class ServiceSearchSteps
 {
-    readonly ConfigModel config;    
-    private readonly string baseUrl;
-    private ServiceSearchRequest request;
-    private HttpResponseMessage lastResponse;
-    private HttpStatusCode statusCode;
-    private const string serviceSearchPath = "api/metrics/service-search";
-    public ServiceSearchSteps()
-    {
-        config = ConfigAccessor.GetApplicationConfiguration();
-        baseUrl = config.BaseUrl;
-    }
+  readonly ConfigModel config;  
+  private readonly string baseUrl;
+  private ServiceSearchRequest request;
+  private HttpResponseMessage lastResponse;
+  private HttpStatusCode statusCode;
+  private const string serviceSearchPath = "api/metrics/service-search";
+  public ServiceSearchSteps()
+  {
+      config = ConfigAccessor.GetApplicationConfiguration();
+      baseUrl = config.BaseUrl;
```

Comment:
refactor: Since we're only using the `BaseUrl` property from `config`, we can remove the `config` member defined on line 19 and just set `baseUrl` as we don't need both

Reply:
Did this and seems its required for bddfy syntax